src/main/java/io/quarkus/bot/MarkClosedPullRequestInvalid.java

```diff
@@ -23,6 +25,18 @@ void markClosedPullRequestInvalid(@PullRequest.Closed GHEventPayload.PullRequest
         GHPullRequest pullRequest = pullRequestPayload.getPullRequest();
 
         if (pullRequest.isMerged()) {
+            if (!pullRequest.getLabels().isEmpty()) {
```

Comment:
You can drop that test given the other advices below (as the log will be inside the logic, you won't need this anymore).